src/src/config/systemMetrics/systemMetrics.ts

```diff
@@ -1,12 +1,60 @@
 import { systemMetricsDictType } from 'types/utils/formatSystemMetricName';
 
 export const systemMetricsDict: systemMetricsDictType = {
-  __system__cpu: 'CPU (%)',
-  __system__p_memory_percent: 'Process Memory (%)',
-  __system__memory_percent: 'Memory (%)',
-  __system__disk_percent: 'Disk (%)',
-  __system__gpu: 'GPU (%)',
-  __system__gpu_memory_percent: 'GPU Memory (%)',
-  __system__gpu_power_watts: 'GPU Power (W)',
-  __system__gpu_temp: 'GPU Temperature (°C)',
+  'system/cpu_utilization_percentage': 'CPU (%)',
+  'system/disk_available_megabytes': 'Disk Available (MB)',
+  'system/disk_usage_megabytes': 'Disk (MB)',
+  'system/disk_usage_percentage': 'Disk (%)',
+  'system/gpu_0_memory_usage_megabytes': 'GPU 0 Memory (MB)',
```

Comment:
Couldn't this be built more cleanly with an iterator?

Reply:
I am not an expert in typescript, so I don't know 😞
I wanted to make sure we can ship this!